master/custom/templates/releasedashboard.html

```diff
@@ -46,4 +46,12 @@ <h4 class="panel-title">
             </div>
         </div>
     </div>
+    <div class="container">
+        <small>Generated at <time id="generatedAt" datetime="{{generated_at}}">{{generated_at}}</time></small>
+    </div>
 </div>
+<script type="javascript">
+    let elem = document.getElementById("generatedAt");
+    let date = new Date(elem.dateTime);
+    elem.innerText = date.toLocaleDateString("en-CA") + " " + date.toLocaleTimeString("pl-PL");
```

Comment:
```suggestion
    elem.innerText = date.toLocaleDateString("en-US") + " (UTC)";
```

Reply:
No, this is invalid.

Reply:
Why is the date formatted twice, in 2 different locales?